File: backend/models/ThuocTinhSanPham.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const ThuocTinhSanPham = sequelize.define(
  "ThuocTinhSanPham",
  {
    id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
    san_pham_id: { type: DataTypes.INTEGER, allowNull: false },
    ten_thuoc_tinh: { type: DataTypes.STRING(100) },
    gia_tri: { type: DataTypes.STRING(255) },
    nhom: { type: DataTypes.STRING(50) },
    thu_tu: { type: DataTypes.INTEGER, defaultValue: 0 },
  },
  {
    tableName: "ThuocTinhSanPham",
    timestamps: false,
  },
);

module.exports = ThuocTinhSanPham;

File: backend-csharp/Models/ThuocTinhSanPham.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class ThuocTinhSanPham
{
    public long Id { get; set; }

    public long SanPhamId { get; set; }

    public string TenThuocTinh { get; set; } = null!;

    public string GiaTri { get; set; } = null!;

    public string? Nhom { get; set; }

    public int ThuTu { get; set; }

    public virtual SanPham SanPham { get; set; } = null!;
}
